docs/src/components/componentList/drawerNavItemComponent/DrawerNavItemExamples.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
There is no need of this component file. You can directly import Markdown file inside the navigation and use it.